jenkins/scripts/bare_metal_lab/cleanup-lab.yaml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: localhost
+  environment:
+    EPHEMERAL_CLUSTER: "{{ EPHEMERAL_CLUSTER }}"
+    EXTERNAL_VLAN_ID: "{{ EXTERNAL_VLAN_ID }}"
+  vars_files:
+    - default_vars/vars.yaml
+  tasks:
+  - name: Cleanup Bare Metal Lab
+    vars:
+      action_deploy: false
```

Comment:
I'm not understanding this variable. What is it for?
When it is set to false, we skip cleaning logs and folders. Also skip cloning the repo?
Maybe it would be better to separate the tasks into different files instead and just include them that we want, instead of disabling tasks based on this variable.

Reply:
removed the variable and sorting tasks with tags now